### ISSUE TITLE
`cmd + shift + G` is used, which is very useful in finder.

### ISSUE THREAD

**Issue by cornradio** [vincelwt/chatgpt-mac#47]

**Comment by nilbus** [comment]
Cmd+shift+g is also useful with text searching where cmd+g finds the next match (for search with cmd+f), and cmd+shift+g finds the previous match. This should either be changed or configurable.

**Comment by vincelwt** [comment]
Yes I agree, which shortcut do you guys think would be best? while still easy to remember

**Comment by cornradio** [comment]
cmd + option + G
how about this one?

**Comment by nilbus** [comment]
that should work

**Comment by felippenardi** [comment]
I stoped using this app because cmd+shift+g is used by Finder, Photoshop, Figma and all other graphic editing softwares to group and ungroup elements.

**Comment by trenkwill** [comment]
I stopped using the app because the shortcut is too important in Figma.

**Comment by esaruoho** [comment]
just wrote a ticket about cmd-shift-g AND cmd-option-p being taken over. glad to see others also find the "Go To..." in Finder to be important to maintain access of..
#72

**Comment by macnason** [comment]
I stopped using the app because it conflicted with Figma. It would be helpful if the keyboard shortcut could be turned off so that it can be remapped through programs like Raycast or Keyboard Maestro.

**Comment by brutesque** [comment]
I would propose a variant on Cmd + Space for spotlight. Perhaps Shift + Command + Space

**Comment by nilbus** [comment]
Other apps also set ctrl shift space, so it would need to be configurable if you want to consider that.

**Comment by johan456789** [comment]
Second it making it configurable.

**Comment by techspertproductdesign** [comment]
Also agree it's a great app but it conflicts with a pretty frequent shortcut for ungrouping in Sketch. Making it configurable has to be the solution to solve the problem.

**Comment by eekwong** [comment]
For now, we have to delete/comment out the GlobalShortcut registration at `/Applications/ChatGPT.app/Contents/Resources/app/index.js`

```    globalShortcut.register("CommandOrControl+Shift+g", () => {
      if (window.isVisible()) {
        mb.hideWindow();
      } else {
        mb.showWindow();
        if (process.platform == "darwin") {
          mb.app.show();
        }
        mb.app.focus();
      }
    });

**Comment by MaximAbdulatif** [comment]
Ooh thank you a million @eekwong that was driving me crazy ! I will start to use the app agin :) 
Cheers from Holland 

Open this file ( by copying the path below and using the useful in Finder ⌘ + Shift+ G 😅 )

/Applications/ChatGPT.app/Contents/Resources/app/index.js


Line 99 changed G to Z and works like a charm :

  globalShortcut.register("CommandOrControl+Shift+z", () => {
